StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/FileHeaderCodeFixProvider.cs

```diff
@@ -352,9 +353,13 @@ private static SyntaxTriviaList CreateNewHeader(string prefixWithLeadingSpaces,
 
         private static string WrapInXmlComment(string prefixWithLeadingSpaces, string copyrightText, string filename, StyleCopSettings settings, string newLineText)
         {
+            string encodedFilename = new XText(filename).ToString();
+            string encodedCompanyName = new XText(settings.DocumentationRules.CompanyName).ToString();
```

Comment:
This doesn't handle `"` characters properly, since it appears in an attribute. It should use the following instead:

```csharp
string encodedFilename = new XAttribute("t", filename).ToString().Substring(2).Trim('"');
string encodedCompanyName = new XAttribute("t", settings.DocumentationRules.CompanyName).ToString().Substring(2).Trim('"');
```